sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -469,19 +472,37 @@ public EncryptedBlobClientBuilder retryOptions(RequestRetryOptions retryOptions)
     }
 
     /**
-     * Sets the {@link HttpPipeline} to use for the service client.
+     * Sets the {@link HttpPipeline} to use for the service client, and adds a decryption policy.
      *
      * If {@code pipeline} is set, all other settings are ignored, aside from {@link #endpoint(String) endpoint}.
```

Comment:
cpk isn't ignored, right? Maybe for a bit of future-proofing, we should specify that any options used to configure a pipeline are ignored?

Reply:
It's also not copied since it isn't a pipeline option. I guess it feels a little weird to stay stuff outside the pipeline won't be copied, but the pipeline is a weird concept for people, so maybe it's best to be explicit.

Reply:
I think they might get confused cause they dont know what options are pipeline and what arent, I'll just say except CPK